lib/ansible/modules/windows/win_optional_feature.ps1

```diff
@@ -0,0 +1,80 @@
+#!powershell
+
+# Copyright: (c) 2019, Carson Anderson <rcanderson23@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#AnsibleRequires -CSharpUtil Ansible.Basic
+
+$spec = @{
+	options = @{
+		name = @{ type = "str"; required = $true }
```

Comment:
You don't have to do it now but accepting a list makes this module a whole lot more useful and faster than just using `with_items`/`loop`.